hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/LambdaTestUtils.java

```diff
@@ -65,6 +66,15 @@ private LambdaTestUtils() {
    */
   public static final String NULL_RESULT = "(null)";
 
+  /**
+   * Atomic references to be used to re-throw an Exception or an ASE
+   * caught inside a lambda function.
+   */
+  public static final AtomicReference<Exception> futureExcp =
```

Comment:
1. not sure that this is the right place for what's been a fairly stable class for a while
1. give full title in their names, upper case
2. make private

Reply:
ah I see, since we wanted the atomic ref to these exceptions for lambda functions, I thought this might be the best place for these, you reckon we move this out of here?

Reply:
yes